### ISSUE TITLE
Show id3 ratings

### ISSUE THREAD

**Issue by dkniffin** [martpie/museeks#268]
These are usually displayed as a set of stars, but internally, I believe id3 stores them as decimals, so we can display them however we want.

**Comment by YurySolovyov** [comment]
I think it clutters the UI so I disable them if I can (in other players), and I'm also not sure we're extracting them ATM.

**Comment by KeitIG** [comment]
Do we consider it as a duplicates of #62 and #101 ?

Integrate a star column does not make any sense if you can't edit the tag directly

**Comment by YurySolovyov** [comment]
Not really #101, I think it is more general.

**Comment by dkniffin** [comment]
Yea, I think it's more general. It does make sense if your tracks all have the rating already, but it certainly will help to be able to edit it.

**Comment by dkniffin** [comment]
We can definitely put it off until after #62 so people who don't use them don't use them can toggle it off.

**Comment by martpie** [comment]
Let's handle that in #401
